### PR TITLE
feat: BREAKING pass URL in email params

### PR DESCRIPTION
**Pull request by Akarshit**
The URL to reset the password is not passed to the mutation instead of being generated by the plugin. This separates the concern between accounts and authentication.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@reactioncommerce/api-plugin-accounts",
   "description": "Accounts plugin for the Reaction API",
-  "version": "0.0.0-development",
+  "version": "1.0.0",
```
Comment (spencern) [inline package.json]: Just in case this one slips the radar - versions should stay at `0.0.0-development`

**Comment by Akarshit** [comment]
Resolved the comments.

**Comment by Akarshit** [comment]
@MohanNarayana Can you please review this one.

**Review by MohanNarayana — APPROVED** [review]

**Pull request merged**